docs/it-manual/api/program-applications.md

```diff
@@ -528,6 +528,13 @@ In addition to the metadata field, file upload questions have the following prop
 - **JSON Type**: `string` or `null`
 - **Format**: A URL, `null` if unanswered.
 - **Description**: A link to the file the applicant uploaded.\
+_Note_: This property is deprecated and soon to be replaced by `file_urls`.
```

Comment:
Should we note that this property is removed when the `MULTIPLE_FILE_UPLOAD_ENABLED` flag is turned on?

Reply:
Done